pkg/apis/monitoring/register.go

```diff
@@ -14,6 +14,9 @@
 
 package monitoring
 
-const (
+// GroupName is set to var instead of const, since this provides the ability for clients importing the module -
+// github.com/prometheus-operator/prometheus-operator/pkg/apis to manage the operator's objects in a different
+// API group
```

Comment:
```suggestion
// GroupName is set to `var` instead of `const`, since this provides the ability for clients importing the module -
// github.com/prometheus-operator/prometheus-operator/pkg/apis to manage the operator's objects in a different
// API group
//
// Use `ldflags` in the client side, e.g.:
// go run -ldflags="-s -X github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring.GroupName=monitoring.example.com" ./example/client/.
```

Reply:
Sure @ArthurSens. Are you suggesting that I add the file with the example too or just the comment?

Reply:
Just the comment is good enough!

Reply:
Thanks @ArthurSens. I dont have the permissions to merge. Could you please merge this?

Reply:
Following up. @ArthurSens @simonpasquier  - Could you pls help get this PR merged? Thanks!